lib/metasploit/framework/data_service/remote/http/remote_host_data_service.rb

```diff
@@ -32,4 +33,20 @@ def update_host(opts)
   def delete_host(opts)
     json_to_mdm_object(self.delete_data(HOST_API_PATH, opts), HOST_MDM_CLASS)
   end
-end
\ No newline at end of file
+
+  def get_host_tags(opts)
+    path = get_path_select(opts, HOST_API_PATH) + '/tag'
```

Comment:
what does `get_path_select` do? you're using it for all your method here but it's only used in the existing methods once

Reply:
Link to the original source code [here](https://github.com/rapid7/metasploit-framework/blob/fc164e0c098456bcc65d3f2605a0580ff0e2965e/lib/metasploit/framework/data_service/remote/http/core.rb#L229), but tl;dr it's a method for adding variables into a URL so that they can be correctly extracted by the `host_servlet`